t/api/02-iso.t

```diff
@@ -27,7 +27,7 @@ use Test::Output qw/stdout_like stderr_like/;
 use OpenQA::Test::Case;
 use OpenQA::Client;
 use Mojo::IOLoop;
-use Data::Dump;
+use Data::Dumper;
```

Comment:
yes, in general. Your commit should just change what is necessary, i.e. don't change 'use Data::Dump' as you don't touch this as @aaannz suggested. I think the `use Mojo::Base -strict` should be kept. It is like `use strict;` The others could be removed if unused now but in another commit.